package.json

```diff
@@ -28,6 +28,7 @@
     "commander": "2.3.0",
     "component-classes": "1.2.1",
     "component-closest": "0.1.4",
+    "component-file-picker": "0.2.1",
```

Comment:
yup, install `clingwrap` and `clingwrap component-file-picker`.

_might_ need to be using `npm@2.14.12` when you do it too? that bit is still unclear to me. @gwwar might be able to shed some light on it.